typescript/sdk/src/token/altVMDeploy.ts

```diff
@@ -49,9 +53,7 @@ export class CosmosNativeDeployer {
       const config = configMapToDeploy[chain];
       assert(config, `No config configured for ${chain}`);
 
-      this.logger.info(
-        `Deploying ${config.type} token to Cosmos Native chain ${chain}`,
-      );
+      this.logger.info(`Deploying ${config.type} token to chain ${chain}`);
 
       switch (config.type) {
         case TokenType.collateral: {
```

Comment:
Here, the typing introduced above makes it easier to see that we are using a token type (`config.token`) instead of a token denom in this call.